aiohttp/connector.py

```diff
@@ -849,7 +849,16 @@ def _make_ssl_context(verified: bool) -> SSLContext:
             sslcontext = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
             sslcontext.options |= ssl.OP_NO_SSLv2
             sslcontext.options |= ssl.OP_NO_SSLv3
-            sslcontext.options |= ssl.OP_NO_COMPRESSION
+            try:
+                sslcontext.options |= ssl.OP_NO_COMPRESSION
+            except AttributeError as attr_err:
+                warnings.warn(
+                    '{!s}: The Python interpreter is compiled '
+                    'against OpenSSL < 1.0.0. Ref: '
+                    'https://docs.python.org/3/library/ssl.html'
+                    '#ssl.OP_NO_COMPRESSION'.
+                    format(attr_err),
+                )
```

Comment:
Why is this in this pull request? Can fixing the travis build and fixing the test failure on OS X be separated into two distinct pull requests?

Reply:
I can't test this w/o the CI config. I think once other comments are settled I could separate it, of course.